src/progressbar/progressbar.ts

```diff
@@ -0,0 +1,34 @@
+import {Component, Input} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-progressbar',
+  template: `
+    <progress class="progress {{striped && 'progress-striped'}} {{type && 'progress-' + type}}" [value]="value" [max]="max">
+      <div class="progress">
+        <span class="progress-bar" [style.width.%]="calculatePercentage()"><ng-content></ng-content></span>
+      </div>
+    </progress>
+  `
+})
+export class NgbProgressbar {
+  private _striped: boolean;
```

Comment:
Yeah, that needs more researching outside this PR because it doesn't work for me out of the box (in fact, I don't see them animated on that page).

So I will open a PR when I figure that out.